wagtail/wagtailadmin/templates/wagtailadmin/pages/confirm_unpublish.html

```diff
@@ -10,7 +10,23 @@
         <form action="{% url 'wagtailadmin_pages:unpublish' page.id %}" method="POST">
             {% csrf_token %}
             <input type="hidden" name="next" value="{{ next }}">
-            <input type="submit" value="{% trans 'Yes, unpublish it' %}" class="button">
+            <ul class="fields">
+                {% if live_descendant_pages %}
```

Comment:
This will call [`QuerySet.__bool__()`](https://github.com/django/django/blob/1.9/django/db/models/query.py#L261-L263), which fetches all of the results before checking if there are any records. This is quite inefficient on large trees. `{% if live_descendant_pages.exists %}` is much more efficient.